docs/core/tools/dotnet-build.md

```diff
@@ -45,11 +45,11 @@ compilation process to be incremental:
 - not load compilation tools from PATH (for example, resgen, compilers)
 - use only known compilers (csc, vbc, fsc)
 
-In order to build an executable application, you need a special configuration section in your project.json file:
+In order to build an executable application instead of a DLL, you need a [special configuration](project-json.md#emitentrypoint) section in your project.json file:
```

Comment:
Thanks for the clarification @blackdwarf. So, do we need also need to change the [emitEntryPoint](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/project-json#emitentrypoint) doc? It says: "true to create an executable; false to produce a .dll file. The default is false." From your explanation, I would get a DLL whether it's true or false.